sources/packages/web/src/services/http/EducationProgramApi.ts

```diff
@@ -133,4 +164,24 @@ export class EducationProgramApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  /**
+   * Education Program Details for ministry users
+   * @param programId program id
+   * @returns Education Program Details
+   */
+  public async getEducationProgramForAEST(
+    programId: number,
+  ): Promise<EducationProgramData> {
+    try {
+      const response = await this.apiClient.get(
```

Comment:
Does API return any error that needs to be shown on toast? if not we can use getCall